### ISSUE TITLE
Plotting issues with upgrade to Mac OS X El Capitan

### ISSUE THREAD

**Issue by GabrielaCR** [GabrielaCR/AGNfitter#4]
After an upgrade to Mac OS X El capitan the code breaks at the plotting stage, due to an issue related to Matplotlib and Latex.  
It looks like El Capitan has made some changes for security, so that the existing latex installs, which probably used /usr/texbin, are no longer allowed, while this should be in /Library/TeX/texbin.

**Comment by GabrielaCR** [comment]
As an easy fix was to install MacTex, in order to have the latex installs in the new correct path in Library. https://tug.org/mactex/
Any better suggestion to improve this is more than welcome!

**Comment by eilatg** [comment]
Does this plotting issue extend to making triangle plots?  I set:

    out['plot_posteriortrianglewithluminosities'] = True  # requires out['calc_intlum']=True

In the SETTINGS file for the example (which runs perfectly fine when it is set to False except for no labels on the plot, see below) and I get the following error running on my Mac (OS Sierra 10.12.6).  

Traceback (most recent call last):
  File "./RUN_AGNfitter_multi.py", line 264, in <module>
    RUN_AGNfitter_multiprocessing(args.ncpu, data_ALL, Modelsdict)
  File "./RUN_AGNfitter_multi.py", line 203, in RUN_AGNfitter_multiprocessing
    catalog_fitting = pool.map(multi_run_wrapper, itertools.izip(range(nsources), itertools.repeat(data_obj), itertools.repeat(modelsdict)))
  File "/Users/eglikman/anaconda2/lib/python2.7/multiprocessing/pool.py", line 253, in map
    return self.map_async(func, iterable, chunksize).get()
  File "/Users/eglikman/anaconda2/lib/python2.7/multiprocessing/pool.py", line 572, in get
    raise self._value
ValueError: Contour levels must be increasing

I checked all the python package versions and those seem fine. My python version is anaconda2-5.0.1

In addition, when running AGNfitter on my Mac, I get output plots that are missing the late that notes XID and redshift. Is this also related?

I have installed the MacTex package as suggested above and there is a symbolic link in /Library/TeX/texbin to the relevant tasks. 

Both appear to work fine on a linux machine that I have tested.  

Any suggestions or thoughts?
